### PR TITLE
CompareEqualZero

### PR DESCRIPTION
**Pull request by echesakov**
https://github.com/dotnet/corefx/issues/26180

### CODE REVIEW

File: src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs [hunk 1]
```diff
@@ -46,6 +46,438 @@ public new abstract class Arm64 : ArmBase.Arm64
...
+            // /// uint64x1_t vcgez_f64 (float64x1_t a)
+            // ///   A64: FCMGE Dd, Dn, #0
+            // /// </summary>
+            // public static Vector64<ulong> CompareGreaterThanOrEqualZero(Vector64<double> value) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// uint16x4_t vcgez_s16 (int16x4_t a)
+            ///   A64: CMGE Vd, Vn, #0
+            /// </summary>
+            public static Vector64<ushort> CompareGreaterThanOrEqualZero(Vector64<short> value) { throw new PlatformNotSupportedException(); }
```
Comment (tannergooding) [inline src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs]: Most of these look to be available on ARM32 as well. `VCGE`, for example.

File: src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs [hunk 2]
```diff
@@ -46,6 +46,438 @@ public new abstract class Arm64 : ArmBase.Arm64
...
+            /// uint32x2_t vcgez_f32 (float32x2_t a)
+            ///   A64: FCMGE Vd, Vn, #0
+            /// </summary>
+            public static Vector64<uint> CompareGreaterThanOrEqualZero(Vector64<float> value) { throw new PlatformNotSupportedException(); }
+
+            // /// <summary>
+            // /// uint64x1_t vcgez_u64 (uint64x1_t a)
+            // ///   A64: CMGE Dd, Dn, #0
+            // /// </summary>
+            // public static Vector64<ulong> CompareGreaterThanOrEqualZero(Vector64<ulong> value) { throw new PlatformNotSupportedException(); }
```
Comment (tannergooding) [inline src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs]: It actually looks like `Vector64<long>/Vector64<ulong` are explicitly reserved today.

**Comment by maryamariyan** [comment]
Thank you for your contribution. As announced in dotnet/coreclr#27549 this repository will be moving to dotnet/runtime on November 13. If you would like to continue working on this PR after this date, the easiest way to move the change to dotnet/runtime is:
1.	In your coreclr repository clone, create patch by running `git format-patch origin`
2.	In your runtime repository clone, apply the patch by running `git apply --directory src/coreclr <path to the patch created in step 1>`

**Comment by maryamariyan** [comment]
Thank you for your contribution. As announced in #27549 the dotnet/runtime repository will be used going forward for changes to this code base. Closing this PR as no more changes will be accepted into master for this repository. If you’d like to continue working on this change please move it to dotnet/runtime.

**Pull request closed without merging**